### PR TITLE
[SPARK-26060][SQL][FOLLOW-UP] Rename the config name.

### PR DESCRIPTION
**Pull request by ueshin**
## What changes were proposed in this pull request?

This is a follow-up of #23031 to rename the config name to `spark.sql.legacy.setCommandRejectsSparkCoreConfs`.

## How was this patch tested?

Existing tests.

### CODE REVIEW

**Comment by ueshin** [comment]
cc @cloud-fan

**Comment by HyukjinKwon** [comment]
retest this please

**Review by HyukjinKwon — APPROVED** [review]

File: sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala [hunk 1]
```diff
@@ -1612,7 +1612,7 @@ object SQLConf {
     .createWithDefault(25)
 
   val SET_COMMAND_REJECTS_SPARK_CONFS =
-    buildConf("spark.sql.legacy.execution.setCommandRejectsSparkConfs")
+    buildConf("spark.sql.legacy.setCommandRejectsSparkConfs")
```
Comment (gatorsmile) [inline sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala]: spark.sql.legacy.setCommandRejectsSparkCoreConfs

**Review by cloud-fan — APPROVED** [review]

**Review by dongjoon-hyun — APPROVED** [review]

**Comment by dongjoon-hyun** [comment]
Retest this please.

**Comment by ueshin** [comment]
Jenkins, retest this please.

**Comment by dongjoon-hyun** [comment]
Thank you all. Merged to master.

**Pull request closed without merging**